internal/daemon/controller/handlers/scopes/scope_service.go

```diff
@@ -72,6 +72,8 @@ var (
 	// TODO: get this from action registry
 	scopeCollectionTypeMapMap = map[string]map[resource.Type]action.ActionSet{
 		scope.Global.String(): {
+			// TODO: uncomment this line.
+			// resource.Alias:            aliases.CollectionActions,
```

Comment:
Should this go in another PR since there should be accompanying changes in scope_service_test.go?